docs/reference/cli/options.md

```diff
@@ -56,6 +56,46 @@ WEB3SIGNER_CONFIG_FILE=/home/me/me_node/config.yaml
   </TabItem>
 </Tabs>
 
+### `commit-boost-api-enabled`
+
+<Tabs>
+  <TabItem value="Syntax" label="Syntax" default>
+
+```bash
+--commit-boost-api-enabled[=<BOOLEAN>]
+```
+
+  </TabItem>
+  <TabItem value="Example" label="Example">
+
+```bash
+--commit-boost-api-enabled=true
+```
+
+  </TabItem>
+  <TabItem value="Environment variable" label="Environment variable">
+
+```bash
+TEKU_COMMIT_BOOST_API_ENABLED=true
+```
+
+  </TabItem>
+  <TabItem value="Configuration file" label="Configuration file">
+
+```bash
+commit-boost-api-enabled: true
+```
+
+  </TabItem>
+</Tabs>
+
+Enables or disables the [Commit-Boost API](https://commit-boost.github.io/commit-boost-client/api).
+The default is `false`.
+
+You can specify proxy keystores for Commit-Boost using
```

Comment:
This needs to be reworded along the lines of:
```
You can specify the location where the generated commit boost proxy keystores will be stored using following option:
[`--proxy-keystores-path`](#proxy-keystores-path)
```. The password file that will be used to encrypted generated proxy keystores can be specified using following option:
[`--proxy-keystores-password-file`](#proxy-keystores-password-file)

Web3Signer process should have read-write access to proxy-keystores-path.